Create 2021-75.md (#280)
JupiterOne/docs: release-notes/2021-75.md

# JupiterOne 2021.75 Release

2021-07-14

## New Features and Improvements

> - Added support for a markdown widget to the Insights app

Reviewer: ```suggestion
- Added a **Markdown widget** to the **Insights** app that allows static HTML content 
  to be added to any Insights board.
```

> - Updated the Owners & Maintainers tab in the Vulnerabilities view to show the query that returned no results if nothing was found

Reviewer: ```suggestion
- Updated the **Owners & Maintainers** tab in the **Vulnerabilities** view to show the 
  query if no results were returned.
```

> - Updated the look & feel of the Policies app

Reviewer: ```suggestion
- Improved UI/UX of the **Policies** app with a more intuitive layout of the 
  domains/policies and the controls/procedures for each domain.
```

> - Customers who have purchased the "Powerup Pack" or are at the Enterprise tier can now upload a custom logo for their J1 instance

Reviewer: ```suggestion
- Customers who have purchased the "Powerup Pack" or are at the Enterprise tier can 
  now upload a **custom logo** for their J1 instance. The custom logo will be
  displayed in the main app UI and used in the policies PDF export.
```

## New Integrations

### Kubernetes Beta

> We have beeen working on an [integration for Kubernetes](https://github.com/JupiterOne/graph-kubernetes/blob/master/docs/jupiterone.md) clusters. [Please reach out](https://forms.gle/HoPsNdtMPwdzVA367) if you're interested in using the integration and help us make it valuable for your operations!

Reviewer: insert blank line after heading. same for the rest of the doc.

### Salesforce Beta
We've been working on an [integration for Salesforce](https://github.com/JupiterOne/graph-salesforce/blob/master/docs/jupiterone.md). The integration currently ingests groups, users, user roles, and permission sets to support user access review and monitoring in JupiterOne. [Please reach out](https://forms.gle/HoPsNdtMPwdzVA367) if you're interested in using the integration. We'd love to hear your thoughts on what to add to make the integration valuable for you.

### Terraform Provider
We're excited to announce the release of our JupiterOne Terraform provider! You can find it in the [Terraform Registry](https://registry.terraform.io/providers/JupiterOne/jupiterone/latest). The provider currently supports managing Questions and Alert Rules. [Let us know](https://forms.gle/HoPsNdtMPwdzVA367) what you'd like to see next!

## Integration Updates

### Artifactory
- Allow user to configure optional ingestion of pipeline resources

### AWS
- Fix AWS API throttling in IAM processing step
- Reduce number of AWS API calls made during IAM policy processing
- Reduce number of AWS API calls made during EC2, Inspector, and ECR processing

> - Improve processing of security groups and build relationships to protected resources (`aws_security_group - PROTECTS -> resource`)

Reviewer 2: should it be `aww_security_group -> PROTECTS -> resource` ?

Author: It renders like that via Preview, not in source

Reviewer 2: it is missing a >

Reviewer 2: or actually looking at previous release notes, it should be `aws_security_group` `PROTECTS` `resouces` to be consistent.

Author: OK, made the changes you've suggested above
